src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentOneDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  const totalPrice = days * rentOneDay;
+
+  if (days >= 3 && days < 7) {
```

Comment:
do we need to do such complicated checks? let's `return` value with the largest discount by default and cases with a basic discount and without it in the conditions, so we can not specify the maximum number of days, and check only on whether it's less than we need or not

also, in this case, it will be much easier for us to expand functionality if we have new rental discounts because we need to add only one simple condition after the others